src/net_processing.cpp

```diff
@@ -1570,7 +1697,7 @@ bool static ProcessHeadersMessage(CNode *pfrom, CConnman *connman, const std::ve
     return true;
 }
 
-bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStream& vRecv, int64_t nTimeReceived, const CChainParams& chainparams, CConnman* connman, const std::atomic<bool>& interruptMsgProc, bool enable_bip61)
+bool static ProcessMessage(PeerLogicValidation* peer_logic, CNode* pfrom, const std::string& strCommand, CDataStream& vRecv, int64_t nTimeReceived, const CChainParams& chainparams, CConnman* connman, const std::atomic<bool>& interruptMsgProc, bool enable_bip61)
```

Comment:
Might be missing something, but can't see any necessity for this new param.

Reply:
I think this is a hold-over from an earlier version of this patch in which some of the data structures were inside PeerLogicValidation -- I think you're right that this should be removed now.

Reply:
good point, it's not needed after latest changes.